doc/js-api/remotestorage.rst

```diff
@@ -16,6 +16,43 @@ example::
      cordovaRedirectUri: 'https://app.wow-much-app.com' // defaults to undefined
    });
 
+Here is a complete list of the default options (set in ``src/config/js``), all
+of which can be overridden by calling the constructor with a configuration
+object like above::
```

Comment:
Why not just extend the object above? It already has comments for the defaults (and much more concise by just having them in the same line with minimal text).